src/input_cp2k_projection_rtp.F

```diff
@@ -55,6 +55,21 @@ SUBROUTINE create_projection_rtp_section(section)
                           repeats=.TRUE.)
 
       NULLIFY (keyword, subsection)
+      CALL keyword_create(keyword, __LOCATION__, name="PROPAGATE_REF", &
+                          description="In case of Ehrenfest dynamics, the atomic basis set is evolving with time. "// &
+                          "The reference MO can either be understood as a spatial-dependent wave-function which is "// &
+                          "time-independent or to be 'attached' with respect to the nuclei position, and "// &
+                          "thus evolve in space as the nuclei moves. For the first case, set this variable to TRUE. "// &
+                          "Note that in this case you shall have enough atomic orbital accross the whole space to "// &
+                          "describe this MO as the nuclei will move and may leave the space where the MO is defined."// &
```

Comment:
The [precommit check](https://storage.googleapis.com/cp2k-ci/run-cp2k-precommit-5c1b6102_report.txt) is failing because there is a space missing after the dot.

Reply:
Are you pointing to "" is defined."// &"" ? In this case, I have just fixed this.